ddp-workspace/projects/ddp-sdk/src/lib/services/irbPassword.service.ts

```diff
@@ -46,7 +46,7 @@ export class IrbPasswordService extends NotAuthenticatedServiceAgent<boolean> {
             // The response body may contain clues as to what went wrong,
             console.error(`Backend returned code ${error.status}, body was: ${error.error}`);
         }
-        return of(false);
+        return throwError(error);
```

Comment:
I bit updated `IrbPasswordService` because components should distinguish cases when a user inputs the wrong password and when a communication error occurred.

Reply:
Sweet!